hack/utils.sh

```diff
@@ -18,7 +18,7 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
 
 # Not from vendor.conf.
 KUBERNETES_VERSION="v1.19.0-beta.2"
-CRITOOL_VERSION=${CRITOOL_VERSION:-baca4a152dfe671fc17911a7af74bcb61680ee39}
```

Comment:
Need to update this line? https://github.com/containerd/containerd/blob/master/script/setup/install-critools#L24

Reply:
thx .. will todo removing this file/normalizing with the new install critools script...